### ISSUE TITLE
Improve Narrator for "Search for Packages" Field In Npm Install Window 

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/nodejstools#1358]
## Repo
1. Enable Windows narrator mode.
2. Open the NTVS npm install window.
3. Wait for the window to load fully.
4. Tab to the npm search input, or trigger this element to be read (one easy way to do this is by enabled mouse mode with caps lock + num lock, and then hovering over the input)
## Expected

The Windows setting search box provides a good example of what should be read. When the empty search box is tabbed to, this is read:

> Search Box; Find a Setting; Editing

When the search box has some text, it reads:

> Search Box; Find a Setting; Editing TEXT

The same is also read in mouse mode.
## Actual

If you tab to the search field in the npm window, the narrator reads:

> Editing.

That's not very helpful.

---

Using mouse mode, if you hover over the empty search input, it says:

> Search for packages; disabled

Closer, but the empty search input should not be marked as disabled.

---

If you enter some text in the search box, using mouse mode it now reads:

> Editable text; TEXT

---

If you tab to the search box, it reads:

> Editing TEXT

**Comment by paulvanbrenk** [comment]
Fixed in Update 3
